### ISSUE TITLE
Jio tv video ads not get blocked by adguard Android 

### ISSUE THREAD

**Issue by spidy2356** [AdguardTeam/AdguardForAndroid#4363]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [yes ] I am running the latest version
- [ yes] I checked the documentation and found no answer
- [ yes] I checked to make sure that this issue has not already been filed

### Issue Details

after recent updates of jio tv, video ads and some time banner ads not get blocked by adguard. Jio tv is a IPTV app, so whenever I change channel it show me a 30sec ad. Sometimes it not showing ads but also streaming is not possible until I change quality of streaming....I don't know why

- **AdGuard version:**
  -  v4.0 nightly 35 (my update channel is release that's why it's 35 not 36)
- **Filtering mode:**
  -  local vpn
- **Device:**
  -  poco x2
- **Operating system and version:**
  -  Android 11 
- **Root access:**
  - no

### Expected Behavior

 whenever I change channel or start a channel in jio tv app there should not be any ads showing

### Actual Behavior

 whenever I start a channel, jio tv app shows 30sec ad every time. And sometimes it doesn't show ads but streaming is also not working when I change quality then it start streaming....I don't know why

### Screenshots

<details><summary>Screenshot:</summary>

Sorry for that...app doesn't allow taking screenshot 
</details>

**Comment by maxikuzmin** [comment]
@spidy2356 check if you have https filtering enabled for this application in Network -> HTTP filtering and try again

**Comment by spidy2356** [comment]
@maxikuzmin I check it with HTTP filtering turn off????......ok let's try..........is there any way to off HTTP filtering for specific app??..... I also add a new issue about Jio TV (issue #4363) but I don't understand what the issue, sometimes ad showing, sometimes not and live stream not play but after changing quality it starts playing. I don't understand is this issue legit or not.......please you check once......if it not legit then close it

**Comment by maxikuzmin** [comment]
@spidy2356 no, you need to check if https filtering is enabled for this app, no need to turn it off. 
It would be better if you also clear the filter log, and try to catch ads in the app so we can see what happens in the filter log. We need screenshots of the filter log as the ads are playing. Also take a screenshot of the ads themselves please. You can send screenshots to devteam@adguard.com

**Comment by spidy2356** [comment]
@maxikuzmin can you explain plz how to check HTTP filter is enabled or not for the app?????.....and for sonyliv app steaming not possible if adblocker is enabled .....so I can give you ss of logs but ads didn't produce it streaming not possible

**Comment by maxikuzmin** [comment]
@spidy2356 you need to go to Settings -> Network -> HTTP filtering then in the applications section check the slider for JioTV.
So when AdGuard is enabled in JioTV, nothing plays?

**Comment by spidy2356** [comment]
> @spidy2356 you need to go to Settings -> Network -> HTTP filtering then in the applications section check the slider for JioTV. So when AdGuard is enabled in JioTV, nothing plays?

no no.......my mistake I think this is my sonyliv app issue......yes everything playing.....but sometimes ads showing.....ok....i check and inform you in few minutes

**Comment by spidy2356** [comment]
@maxikuzmin I didn't fount anything in setting like you said.... settings> network > HTTP filtering.......so I just cleared the filter log and open the app for ad production but in the small time gap, there is so many filter logs generated so how I manage to take  screenshot and send you????

**Comment by maxikuzmin** [comment]
@spidy2356 close all apps, clear filter log and repeat steps to retry ads. You can send multiple screenshots at once. 
I'm record instructions for https filtering. Try turning it on and trying again

https://user-images.githubusercontent.com/101641415/214390645-8ff73067-5081-4098-a9d4-988faf421ff7.mp4

**Comment by spidy2356** [comment]
![Screenshot_2023-01-25-01-08-20-778_com adguard android](https://user-images.githubusercontent.com/101243689/214392010-b05db488-88aa-42e2-b849-127c4f2d0025.jpg)
I have only this options.......not like yours
![Screenshot_2023-01-25-01-12-06-124_com adguard android](https://user-images.githubusercontent.com/101243689/214392710-7c0ba06e-968e-4b34-824d-1f78d9761994.jpg)

**Comment by maxikuzmin** [comment]
@spidy2356 you wrote in Issue details v4.0 nightly 35. Okay, if we talk about version 3.6.10-11, 
here's what you need to do:

https://user-images.githubusercontent.com/101641415/214394321-d7e93194-bc0c-4c80-958f-e0abece80e03.mp4

**Comment by spidy2356** [comment]
@maxikuzmin it's not possible,,,,,,,,after turn on https filtering for jio tv the app doesn't access internet....so I must be take ss of filter logs after uncheck https filtering for jio tv

**Comment by spidy2356** [comment]
![Screenshot_2023-01-25-01-29-53-450_com mi android globallauncher](https://user-images.githubusercontent.com/101243689/214396252-c79ac5da-1633-4d61-a33d-22c55e1945b3.jpg)

**Comment by maxikuzmin** [comment]
@spidy2356 okay, then waiting for screenshots of the filtering log. Reset https filtering, turn it off

**Comment by spidy2356** [comment]
@maxikuzmin this provided ss is the case of "change video quality then playback start".... showing ads case not occur everytime.... specially occurs for live match....so 2nd case ss will upload tomorrow
![Screenshot_2023-01-25-01-38-20-674_com adguard android](https://user-images.githubusercontent.com/101243689/214398185-a6c3ce24-ab7f-49cf-b98f-8a854a9d5892.jpg)
![Screenshot_2023-01-25-01-38-20-674_com adguard android](https://user-images.githubusercontent.com/101243689/214398286-88a5e33d-8dfe-44b7-a9fc-e6318a4126d6.jpg)

**Comment by maxikuzmin** [comment]
@spidy2356 well, I asked to send to email devteam@adguard.com, but okay, we'll check them here :) 
Please send screenshots of the ads to devteam@adguard.com, it will be more convenient later

**Comment by spidy2356** [comment]
@maxikuzmin I applied for beta tester program 2 or 3 weeks ago but didn't get reply or email yet......what to do now???....and I send the app screenshot tomorrow

**Comment by maxikuzmin** [comment]
@spidy2356 please duplicate the email to [devteam@adguard.com](mailto:devteam@adguard.com). I need your email and I'll send you the key for beta testing.
I'm closing this issue as you asked in the other issue

**Comment by spidy2356** [comment]
> @spidy2356 please duplicate the email to [devteam@adguard.com](mailto:devteam@adguard.com). I need your email and I'll send you the key for beta testing. I'm closing this issue as you asked in the other issue

thanks bro 😊😊
